website/docs/r/sagemaker_endpoint_configuration.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "aws"
+page_title: "AWS: aws_sagemaker_endpoint_configuration"
+sidebar_current: "docs-aws-resource-sagemaker-endpoint-configuration"
+description: |-
+  Provides a SageMaker endpoint configuration resource.
+---
+
+# aws_sagemaker_endpoint_configuration
+
+Provides a SageMaker endpoint configuration resource.
+
+## Example Usage
+
+
+Basic usage:
+
+```hcl
+resource "aws_sagemaker_endpoint_configuration" "ec" {
+    name = "my-endpoint-config"
+
+    production_variant {
+        variant_name            = "variant-1"
+        model_name              = "${aws_sagemaker_model.m.name}"
+        initial_instance_count  = 1
+        instance_type           = "ml.t2.medium"
+    }
+
+    tags {
+        Name = "foo"
+    }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Optional) The name of the endpoint configuration. If omitted, Terraform will assign a random, unique name.
+* `production_variants` - (Required) Fields are documented below.
+* `kms_key_id` - (Optional) KMS key to encrypt the model data.
+* `tags` - (Optional) A mapping of tags to assign to the resource.
+
+The `production_variant` block supports:
+
+* `variant_name` - (Optional) The name of the variant. If omitted, Terraform will assign a random, unique name.
```

Comment:
Nit: These attributes should be alphabetically sorted either as a whole or required first then optional